javaagent-tooling/src/test/groovy/io/opentelemetry/javaagent/test/DefaultInstrumenterTest.groovy

```diff
@@ -93,9 +93,8 @@ class DefaultInstrumenterTest extends AgentSpecification {
 
   def "configure default sys prop as #value"() {
     setup:
-    ConfigUtils.updateConfig {
```

Comment:
Is this attributed to me because I merged the PR? This was @tylerbenson's work.

Reply:
Oh no, you're right! I cherry-picked the merge commits which is why they're attributed this way. I'll fix these up (we preserve these commits on the dd-trace-java merge PRs as opposed to our default squash merge).